index.html

```diff
@@ -0,0 +1,87 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <title>Door Status</title>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
```

Comment:
Please use libjs-jquery from apt. This prevents external requests (which may be blocked by the user agent) and therefore also increases privacy. Also, the apt package gets security updates.

Reply:
but then i would have to move this to an ansible role and it would not be static site anymore